optimum/quanto/tensor/qtensor_func.py

```diff
@@ -68,85 +66,3 @@ def has_compatible_shallow_copy_type(func, input: torch.Tensor, from_: torch.Ten
 )
 def unsupported_op(func, *args, **kwargs):
     return qfallback(func, *args, **kwargs)
-
```

Comment:
Not sure why you moved this but it has to be refactored anyway.

Reply:
It was due to the 20% latency overhead we discussed above.